packages/types/src/element/productElement.ts

```diff
@@ -18,7 +18,25 @@
  * Internal dependencies
  */
 import type { Element } from './element';
-
-export interface ProductElement extends Element {
+interface ProductImage {
+  alt: string;
+  url: string;
+}
+export interface Product {
   productId: string;
+  productBrand: string;
+  productDetails: string;
+  productImages: ProductImage[];
+  productPrice: number;
+  productPriceCurrency: string;
+  productTitle: string;
+  productUrl: string;
+}
+export interface ProductElement extends Element {
+  product: Product;
+}
+export interface ShoppingAttachmentType {
```

Comment:
We don't usually suffix types with "Type".

```suggestion
export interface ShoppingAttachment {
```

Reply:
Fixed in [0f952b7](https://github.com/GoogleForCreators/web-stories-wp/pull/12382/commits/0f952b7b62092961c0d19a90db729dec46711097)